spectrum/autism_prevalence_map/management/commands/pull_studies_from_gsheets.py

```diff
@@ -44,49 +47,46 @@ def load_research_data(self):
+                        'samplesize': data['c'][4]['v'] if data['c'][4] is not None else '', 
+                        'age': data['c'][5]['v'] if data['c'][5] is not None else '', 
+                        'individualswithautism': data['c'][6]['v'] if data['c'][6] is not None else '', 
+                        'diagnosticcriteria': data['c'][7]['v'] if data['c'][7] is not None else '', 
+                        'diagnostictools': data['c'][8]['v'] if data['c'][8] is not None else '', 
+                        'percentwaverageiq': data['c'][9]['v'] if data['c'][9] is not None else '', 
+                        'sexratiomf': data['c'][10]['v'] if data['c'][10] is not None else '', 
+                        'prevalenceper10000': data['c'][11]['v'] if data['c'][11] is not None else '', 
+                        'confidenceinterval': data['c'][12]['v'] if data['c'][12] is not None else '', 
+                        'categoryadpddorasd': data['c'][13]['v'] if data['c'][13] is not None else '',
+                        'yearsstudied': data['c'][14]['v'] if data['c'][14] is not None else '',
+                        'recommended': data['c'][15]['v'] if data['c'][15] is not None else '', 
+                        'studytype': data['c'][16]['v'] if data['c'][16] is not None else '',
+                        'meanincomeofparticipants': data['c'][17]['v'] if data['c'][17] is not None else '',
+                        'educationlevelofparticipants': data['c'][18]['v'] if data['c'][18] is not None else '',
+                        'citation': data['c'][19]['v'] if data['c'][19] is not None else '',
+                        'link1title': data['c'][20]['v'] if data['c'][20] is not None else '',
+                        'link1url': data['c'][21]['v'] if data['c'][21] is not None else '',
+                        'link2title': data['c'][22]['v'] if data['c'][22] is not None else '',
+                        'link2url': data['c'][23]['v'] if data['c'][23] is not None else '',
+                        'link3title': data['c'][24]['v'] if data['c'][24] is not None else '',
+                        'link3url': data['c'][25]['v'] if data['c'][25] is not None else '',
+                        'link4title': data['c'][26]['v'] if data['c'][26] is not None else '',
+                        'link4url': data['c'][27]['v'] if data['c'][27] is not None else ''
                     }
-                    obj, created = studies.objects.update_or_create(gsheet_id=data['id']['$t'], defaults=updated_values)
+                    obj, created = studies.objects.update_or_create(gsheet_id=index, defaults=updated_values)
 
 
             except Exception as e:
                 # if error
+                print(index)
                 print('load research data error')
                 print(e)
 
     def geocode(self):
-        gmaps_api_key = '&key=' + 'AIzaSyACddN3i59_QccZTqB4cWGyK6ZDFCLCVBE'
+        gmaps_api_key = '&key=AIzaSyACddN3i59_QccZTqB4cWGyK6ZDFCLCVBE'
```

Comment:
I think this was concatenated bc we had it in mind to have this be an environment variable. We should revisit that; that said, this is a very sensible change